pkg/controller/prpqr_reconciler/pjstatussyncer/pjstatussyncer.go

```diff
@@ -0,0 +1,122 @@
+package pjstatussyncer
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/apimachinery/pkg/types"
+	prowv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
+	controllerruntime "sigs.k8s.io/controller-runtime"
+	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+
+	prpqv1 "github.com/openshift/ci-tools/pkg/api/pullrequestpayloadqualification/v1"
+)
+
+const (
+	controllerName = "prowjob_status_syncer"
+)
+
+func AddToManager(mgr controllerruntime.Manager, ns string) error {
+	ctrl, err := controller.New(controllerName, mgr, controller.Options{
+		MaxConcurrentReconciles: 10,
+		Reconciler: &reconciler{
+			logger: logrus.WithField("controller", controllerName),
+
+			client: mgr.GetClient(),
+		},
+	})
+	if err != nil {
+		return fmt.Errorf("failed to construct controller: %w", err)
+	}
+
+	// Watch only on updates
+	predicateFuncs := predicate.Funcs{
+		CreateFunc: func(e event.CreateEvent) bool { return false },
+		DeleteFunc: func(event.DeleteEvent) bool { return false },
+		UpdateFunc: func(e event.UpdateEvent) bool {
+			if _, ok := e.ObjectNew.GetLabels()["pullrequestpayloadqualificationruns.ci.openshift.io"]; !ok {
```

Comment:
extract the label name to a const

Reply:
Yea this most probably will be shared with the other components.